docs/development.md

```diff
@@ -0,0 +1,181 @@
+# Development
+
+## Structure
+
+This repository is a monorepo. Here's the list of all the apps and packages in it (each app has its own README):
+
+### Apps
+
+- [`apps/storefront`](../apps/storefront/README.md): Next.js e-commerce storefront starter pack
+- [`apps/saleor-app-checkout`](../apps/saleor-app-checkout/README.md): a Next.js Saleor app with a dashboard for managing settings and theme, backend for Checkout, and the Checkout Storefront – ready to be extended/modified
+
+### Packages
+
+- `packages/ui-kit`: UI kit for Storefront and Checkout
+- `packages/config-checkout`: `eslint` configuration
+- `packages/config-storefront`: `eslint` configuration
+- `packages/tsconfig`: `tsconfig.json`s used throughout the monorepo
+- `packages/checkout-storefront`: Checkout storefront component
+- `packages/checkout-common`: Common types and utils
+
+## Getting started
+
+### Prerequisites
+
+This monorepo uses `pnpm` as a package manager, so you will have to install it beforehand. To install it, run:
+
+```bash
+npm install -g pnpm
+```
+
+### Installing dependencies
+
+To install dependencies, run:
+
+```bash
+pnpm i
+```
+
+### Running
+
+To start all the dev servers, run:
+
+```bash
+pnpm dev
+```
+
+[The command will boot up 4 applications running on different ports.](#ports).
+
+You can also start a specific app by running:
+
+```bash
+pnpm dev --filter=saleor-app-checkout
+```
+
+#### [`saleor-app-checkout`](../apps/saleor-app-checkout/README.md)
+
+First, you need to change the default environment variables. Create `.env.local` file in `apps/saleor-app-checkout`, as described [here](../apps/saleor-app-checkout/README.md#env-variables).
```

Comment:
This one is outdated - you can create a single `.env.local` file at the root of monorepo.

I think we should put the "Env vars" paragraph here instead. In practice, you only need to change the Saleor endpoint in `SALEOR_API_URL`. The rest can stay the same for development.

Reply:
So there is nothing you need to do with env vars specifically for `saleor-app-checkout` to work? It's a general instruction for all the apps?